components/conjunctions/_base.scss

```diff
@@ -14,19 +14,19 @@ ul[class*="cnjnctn-type-"] {
 	position: relative;
 
 	&.brdr-0 {
-		[class*="cnjnctn-col"]:after{
+		> [class*="cnjnctn-col"]:after{
```

Comment:
Since there is only one rule here, move to the rule above so it should be `&.brdr-0 > [class*="cnjnctn-col"]:after {`